src/components/DepositWidget/Styled.tsx

```diff
@@ -38,53 +27,49 @@ export const InnerWrapper = styled.div`
     text-align: center;
   }
 
-  ul {
+  .WalletItemContainer {
+    display: grid;
+    grid-template-rows: repeat(2, auto) 20px auto;
```

Comment:
`repeat(2, auto) 20px auto` vs `auto auto 20px auto`
I would understand if it was `repeat(10` or `repeat(2, auto 20 px 1 fr <complex pattern>)`
Not that it really matters

Reply:
It used to be repeat 4 and I was lazy ;)